services/web/server/src/simcore_service_webserver/wallets/api.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from aiohttp import web
+from models_library.api_schemas_webserver.wallets import WalletGet, WalletGetPermissions
+from models_library.users import UserID
+from models_library.wallets import UserWalletDB, WalletID
+
+from . import _db as db
+from .errors import WalletAccessForbiddenError
+
+log = logging.getLogger(__name__)
+
+
+async def get_wallet_by_user(
```

Comment:
in the public `wallets.api` is a public module. I would still keep the implementation of these functions in `_api` and just import + expose them here (i.e. with `__all__`). The advantage is that you control exactly what you expose to this public API. For instance, this way you do not inadvertaly expose things like `wallet.db` as you are doing here since I can``from wallets.api import db``

Reply:
good tip, thanks!